rust/lance-file/src/v2/writer.rs

```diff
@@ -338,10 +338,10 @@ impl FileWriter {
         self.write_pages(encoding_tasks).await?;
 
         // No data, so don't create a file
-        if self.rows_written == 0 {
-            self.writer.shutdown().await?;
-            return Ok(0);
-        }
+        // if self.rows_written == 0 {
+        //     self.writer.shutdown().await?;
+        //     return Ok(0);
+        // }
```

Comment:
You can go ahead and remove these lines.  It might be good to add a test but I can do that later too.